channels/lb.m3u

```diff
@@ -10,8 +10,6 @@ https://manar.live/x.smil/tracks-v1a1/mono.m3u8
 https://lmdstrm.cdn.octivid.com/mayadeen-live/smil:mayadeen.smil/playlist.m3u8
 #EXTINF:-1 tvg-id="AlhayatTV.lb" tvg-name="Alhayat TV" tvg-country="LB" tvg-language="Arabic" tvg-logo="https://www.alhayat.tv/images/logo/main.34f5c635.png" group-title="General",Alhayat TV (720p)
 https://wowzaprod140-i.akamaihd.net/hls/live/750788/cdb837ca/playlist.m3u8
```

Comment:
```suggestion
https://wowzaprod140-i.akamaihd.net/hls/live/750788/cdb837ca/playlist.m3u8
#EXTINF:-1 tvg-id="AlimanTV.lb" tvg-name="Aliman TV" tvg-country="LB" tvg-language="Arabic" tvg-logo="https://i.imgur.com/817xRPg.png" group-title="Religious",Aliman TV (240p) [Not 24/7]
https://svs.itworkscdn.net/alimanlive/imantv.smil/playlist.m3u8
```

Reply:
Guys, it has changed...
https://svs.itworkscdn.net/alimanlive/imantv.smil/alimanpublish/imantv/chunks.m3u8

Reply:
Works fine https://streamtest.in/logs/https-svs-itworkscdn-net-alimanlive-imantv-smil-alimanpublish-imantv-playlist-m3u8